### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by degisew**
### Bookstore: initialize the project with components
**In this PR, I implemented the following features:**

- Initialised React app.
- Created a directory for your components.
- Created reusable components for the building blocks of the app. 
- Added [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
- Created the book view that displays the list of books.
- Created a component called Book for displaying a single book.
- Created a form for adding a book (no functionality yet).
- Created a separate component for this form.
- Categories:
        - Should display a button "Check status" only.

### CODE REVIEW

File: src/Components/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function AddBook() {
+  return (
+    <div>
+      <form action="#" className="form">
+        <input type="text" />
+        <button type="button">Add</button>
+      </form>
+    </div>
```
Comment (KDlamini) [inline src/Components/AddBook.js]: The inputs for the add book form should be title and author as per project requirements.

image

- Kindly include the input field for the author and make sure it is displayed according to the design layout.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @degisew 🙋‍♀️,  

Great work so far! 💯
You have made a great effort in initializing the Bookstore project with components.
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Create-react-app is used to initialize React app
✔️  React Router used for navigation
✔️  Correct PR title and description
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/AddBook.js (hunk 1 above)
Comment (degisew) [reply src/Components/AddBook.js]: Okay, thanks. I will add it.

**Review by jaflih — APPROVED** [review]
Hi @degisew  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
Linters working properly ✔️
Commit messages are meaningful ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**